Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -286,13 +286,16 @@ protected override int CreateToolWindow(ref Guid toolWindowType, int id)
 
         private static void ProfileStandaloneTarget(SessionNode session, StandaloneTarget runTarget, bool openReport) {
             LaunchConfiguration config;
-            if (runTarget.PythonInterpreter != null) {
+            if (runTarget.PythonInterpreter != null && runTarget.PythonInterpreter.Id != "Global|Other...") {
```

Comment:
Anything in `Strings.` gets localized, which means `Other` might not be the right thing to compare against. You should be using `Strings.LaunchProfiling_OtherInterpreter` for the second part of the string, like you did in `StandaloneTargetView.cs`